news/497.feature

```diff
@@ -0,0 +1,2 @@
+Switch to 6.docs.plone.org (was 6.dev-docs.plone.org)
```

Comment:
I would argue that this is not a new feature, but that's the best match out of what is available in the towncrier configuration.

I wish we had a better name for documentation changes.

https://github.com/plone/plone.api/blob/master/pyproject.toml#L12-L16

Should I make a new issue to request that addition for a future PR, not this one?

Reply:
Thought the same.

Would be nice to have a towncrier news entry type '.doc'.
I think plone.releaser is the place to suggest an addition. See https://github.com/plone/plone.releaser/blob/master/ADD-A-NEWS-ITEM.rst

Reply:
Each project that uses `towncrier` has a `pyproject.toml` where the known suffixes are defined. For packages like `plone.api` from which we include the documentation on docs.plone.org, it makes sense to allow `.doc`. Maybe for all packages.
I will merge this PR, then copy the `pyproject.toml` from `plone.restapi` and fix the news snippet extension.

Reply:
I would prefer `.documentation` because PyCharm and macOS attempt to open `.doc` in Word or other Office app.

Reply:
done